src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCheckSwitchExpressionDeclaration.java

```diff
@@ -0,0 +1,28 @@
+//non-compiled with javac: Compilable with Java14                                   //indent:0 exp:0
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;             //indent:0 exp:0
+                                                                                  //indent:82 exp:82
+/* Config:                                                                          //indent:0 exp:0
+ *                                                                                  //indent:1 exp:1
+ * basicOffset = 4                                                                  //indent:1 exp:1
+ * braceAdjustment = 0                                                              //indent:1 exp:1
+ * caseIndent = 4                                                                   //indent:1 exp:1
+ * throwsIndent = 4                                                                 //indent:1 exp:1
+ * arrayInitIndent = 4                                                              //indent:1 exp:1
+ * lineWrappingIndentation = 8                                                      //indent:1 exp:1
+ * forceStrictCondition = false                                                     //indent:1 exp:1
+ */                                                                                 //indent:1 exp:1
+public class InputIndentationCheckSwitchExpressionDeclaration {                     //indent:0 exp:0
+                                                                                  //indent:82 exp:82
+    public static void main(String... args) {                                       //indent:4 exp:4
+        if (args.length < 2) {                                                      //indent:8 exp:8
+            return;                                                               //indent:12 exp:12
+        }                                                                           //indent:8 exp:8
+        String requestId = switch (args[0]) {                                       //indent:8 exp:8
+            case "CREATE" -> "RANDOM";                                            //indent:12 exp:12
+                case "UPDATE" -> args[1];                                    //indent:16 exp:12 warn
+                default -> "DEFAULT";                                        //indent:16 exp:12 warn
```

Comment:
I spotted minor message inaccuracy. `23:17: 'case' child has incorrect indentation level 16, expected level should be 12.`
I think better would be to `23:17: 'default' child has incorrect indentation level 16, expected level should be 12.`
Caused by CaseHandler processing both `case` and `default` keywords: [source](https://github.com/checkstyle/checkstyle/blob/13481f2c410e4944ecf5ab93ec49948a523a0c82/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/CaseHandler.java#L34-L37)
But always using string constant `case` as `typeName` (used to format error message): [source](https://github.com/checkstyle/checkstyle/blob/13481f2c410e4944ecf5ab93ec49948a523a0c82/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/CaseHandler.java#L49)
Not sure if its important or not.

Reply:
I noticed the same, please open a new issue for further discussion.

Reply:
https://github.com/checkstyle/checkstyle/issues/10416

Reply:
Resolved.